src/t_stream.c

```diff
@@ -1422,39 +1412,56 @@ int streamRangeHasTombstones(stream *s, streamID *start, streamID *end) {
     return 0;
 }
 
-/* Replies with a consumer group's current lag, that is the number of messages
- * in the stream that are yet to be delivered. In case that the lag isn't
- * available due to fragmentation, the reply to the client is a null. */
+/* Replies with a consumer group's current lag, which is the number of messages in the stream
+ * that are yet to be delivered. Additionally, it includes an entries-read field that indicates
+ * the number of messages currently read. In case that the lag or entries-read isn't available
+ * due to fragmentation, the reply to the client is null. */
 void streamReplyWithCGLag(client *c, stream *s, streamCG *cg) {
```

Comment:
I think we should update the method naming appropriately. maybe `streamReplyWithCGLagAndEntriesRead`